### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by BALAJI141031**
responsive navigation added for all pages
files to check:navigation

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -120,49 +105,71 @@ body{
...
+  display: none;
+}
+
+.mobile-sidebar {
+  position: absolute;
+  z-index: 1;
+  background-color: var(--span-color);
+  padding: 1rem 5rem 1rem 10px;
+  /* height: 60vh; */
+  font-size: 24px;
```
Comment (mayuresh-tech) [inline css/main.css:118]: Remove commented code

File: pages/components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  align-items: flex-start;
+}
+
+/* .sidebar {
+  display: none;
+} */
+
+/* .mobile-sidebar {
+  position: absolute;
+  z-index: 1;
```
Comment (mayuresh-tech) [inline pages/components/navigation/navigation.css:24]: Same here
Try to avoid commented code

File: pages/components/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    <link
+      rel="stylesheet"
+      href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css"
+    />
+    <script src="../../../index.js" defer></script>  
+    <title>Navigation</title>
+  </head>
+  <body>   
+    <!-- desktop header -->
+    <header class="desktop-nav">
```
Comment (mayuresh-tech) [inline pages/components/navigation/navigation.html:25]: Good explaination

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

**Review by dipen2000 — APPROVED** [review]
Overall no problem others have already mentioned some suggestions so look over that otherwise this is good to go.

**Pull request merged**